.travis.yml

```diff
@@ -1,50 +0,0 @@
-language: php
-
-php:
-  - 7.2
-  - 7.3
-  - 7.4
-
-env:
-  matrix:
-    - DB=mysql db_dsn='mysql://root@127.0.0.1/cakephp_test'
-    - DB=pgsql db_dsn='postgres://postgres@127.0.0.1/cakephp_test'
-    - DB=sqlite db_dsn='sqlite:///:memory:'
-  global:
-    - DEFAULT=1
-
-services:
-  - mysql
-  - postgresql
-
-matrix:
-  fast_finish: true
-  include:
-    - php: 7.2
-      env: PHPCS=1 DEFAULT=0
-    - php: 7.2
-      env: STATIC_ANALYSIS=1 DEFAULT=0
-    - php: 7.2
-      env: PREFER_LOWEST=1
-
-install:
-  - if [[ $PREFER_LOWEST != 1 ]]; then composer install --prefer-source --no-interaction ; fi
-  - if [[ $PREFER_LOWEST == 1 ]]; then composer update --prefer-dist --no-interaction --prefer-lowest --prefer-stable ; fi
-
-before_script:
-  - if [[ $DB = 'mysql' ]]; then mysql -e 'CREATE DATABASE cakephp_test;'; fi
-  - if [[ $DB = 'pgsql' ]]; then psql -c 'CREATE DATABASE cakephp_test;' -U postgres; fi
-  - if [[ $STATIC_ANALYSIS == 1 ]]; then composer stan-setup; fi
-
-script:
-  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION = 7.3 ]]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover=clover.xml; fi
-  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION != 7.3 ]]; then vendor/bin/phpunit; fi
-
-  - if [[ $PHPCS = 1 ]]; then vendor/bin/phpcs -n -p src/ tests/; fi
-  - if [[ $STATIC_ANALYSIS = 1 ]]; then composer stan; fi
-
-after_success:
-  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION = 7.3 ]]; then bash <(curl -s https://codecov.io/bash); fi
-
-notifications:
-  email: false
```

Comment:
Refs https://github.com/cakephp/authentication/pull/419#pullrequestreview-526926081

Reply:
The email notification?

Reply:
Wrong line I guess, no, this was referring to the dist XML file to best be kept.

Reply:
I didn't change phpunit.xml.dist.

Reply:
OK, then all good.